metric-schema-java/src/integrationInput/java/com/palantir/test/NamespaceTagsMetrics.java

```diff
@@ -0,0 +1,196 @@
+package com.palantir.test;
+
+import com.codahale.metrics.Meter;
+import com.google.errorprone.annotations.CheckReturnValue;
+import com.palantir.logsafe.Preconditions;
+import com.palantir.logsafe.Safe;
+import com.palantir.tritium.metrics.registry.MetricName;
+import com.palantir.tritium.metrics.registry.TaggedMetricRegistry;
+import java.util.Objects;
+
+/**
+ * General web server metrics.
+ */
+public final class NamespaceTagsMetrics {
+    private static final String JAVA_VERSION = System.getProperty("java.version", "unknown");
+
+    private static final String LIBRARY_NAME = "witchcraft";
+
+    private static final String LIBRARY_VERSION =
+            Objects.requireNonNullElse(NamespaceTagsMetrics.class.getPackage().getImplementationVersion(), "unknown");
+
+    private final TaggedMetricRegistry registry;
+
+    private final String noValueTag;
+
+    private final NamespaceTags_LocatorWithMultipleValues locatorWithMultipleValues;
+
+    private NamespaceTagsMetrics(
+            TaggedMetricRegistry registry,
+            String noValueTag,
+            NamespaceTags_LocatorWithMultipleValues locatorWithMultipleValues) {
+        this.registry = registry;
+        this.noValueTag = noValueTag;
+        this.locatorWithMultipleValues = locatorWithMultipleValues;
+    }
+
+    public static NamespaceTagsMetrics of(
+            TaggedMetricRegistry registry,
+            String noValueTag,
+            NamespaceTags_LocatorWithMultipleValues locatorWithMultipleValues) {
+        return new NamespaceTagsMetrics(
+                Preconditions.checkNotNull(registry, "TaggedMetricRegistry"),
+                Preconditions.checkNotNull(noValueTag, "noValueTag"),
+                Preconditions.checkNotNull(locatorWithMultipleValues, "locatorWithMultipleValues"));
+    }
```

Comment:
Not sure if you're already tracking, but I think we should make this a staged builder, otherwise tags with the same type will be easy to order incorrectly (same reason we use staged builders for individual metric builders)

Reply:
Yes, just wanted to get the simple thing first. I'll get validation and the staged builder done next. Reason I hesitated is just that there's already a builder generator, so probably will have to refactor some things first

Reply:
Alright we gotta builder!